NguyenVanViet2004/FE-GlowUp
# Improve UI elements optimize code for map step details

## Related issues
Resolved https://github.com/NguyenVanViet2004/FE-GlowUp/issues/53

## Summary
- Rename `ServiceDetailTemplate` to `StepDetails`
- Improving UI elements, optimize logic code
- Rename header title for combo/ComboDetails & map/Map

## How to test
- Run the app without errors
- Check step details and ensure the step is display correct

## Concern points

## Screenshots
| --- | --- |
| --- | --- |
|![Simulator Screenshot - iPhone 16 Plus - 2024-11-23 at 17 17 38](https://github.com/user-attachments/assets/b6fe7685-364c-428b-bdbd-49e096a41dc8)|![Simulator Screenshot - iPhone 16 Plus - 2024-11-23 at 17 17 42](https://github.com/user-attachments/assets/ab2e4f49-b8c4-441f-96e4-f09827b7612a)|
